docs/guides/triggering-pipelines.mdx

```diff
@@ -62,6 +62,13 @@ payload.
 * (Schedule triggers only) Skip run if previous run still in progress
   * Enabling this setting will tell the scheduler to not initialize the current run if any
     previous run for this trigger is still in progress.
+* (Schedule triggers only) [available starting in v0.9.60] Create initial pipeline run if start date is before current execution period
```

Comment:
This might be changed to v0.9.61 since this feature may not be included in the very next release.